test/units/galaxy/test_collection.py

```diff
@@ -198,6 +199,48 @@ def manifest(manifest_info):
             yield fake_file, sha256(b_data).hexdigest()
 
 
+@pytest.fixture()
+def server_config(monkeypatch):
+    monkeypatch.setattr(C, 'GALAXY_SERVER_LIST', ['server1', 'server2', 'server3'])
+
+    default_options = dict((k, None) for k, v in SERVER_DEF)
```

Comment:
FYI dict comprehensions exist since Python 2.7:
```suggestion
    default_options = {k: None for k in SERVER_DEF.keys()}
```

Reply:
OTOH this is probably more elegant:
```suggestion
    default_options = dict.fromkeys(SERVER_DEF.keys())
```

Reply:
Oh, undoing this, it's not a dictionary